### ISSUE TITLE
Throws binary incompatability error when importing on python 3.8

### ISSUE THREAD

**Issue by sorenmulli** [wikipedia2vec/wikipedia2vec#66]
Hello, I get the following output when attempting to import wikipedia2vec on python 3.8

```py
[I] ➜ ipython            
Python 3.8.7 (default, Feb 26 2021, 07:41:46) 

In [1]: import wikipedia2vec
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-1-466a1980f148> in <module>
----> 1 import wikipedia2vec

~/.pyenv/versions/luke38/lib/python3.8/site-packages/wikipedia2vec/__init__.py in <module>
      2 # License: Apache License 2.0
      3 
----> 4 from .dictionary import Dictionary
      5 from .wikipedia2vec import Wikipedia2Vec

__init__.pxd in init wikipedia2vec.dictionary()

ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject
```

Info:
Python: 3.8.7
Numpy: 1.19.5

**Comment by ikuyamada** [comment]
Hi,
This error may be due to the mismatch between NumPy used to compile Wikipedia2Vec and Numpy that is currently installed. Does reinstalling NumPy and Wikipedia2Vec fix the error?

**Comment by sorenmulli** [comment]
Reinstalling actually fixes the error if I install numpy 1.20.
But on numpy 1.19 I get the above error even if reinstall.
I am using 1.19 as this is the dependency of Tensorflow.

**Comment by ikuyamada** [comment]
I think the error is fixed if you compile Wikipedia2Vec using numpy 1.19. Please Install Numpy 1.19 first, and then reinstall Wikipedia2Vec.

**Comment by sorenmulli** [comment]
Yay! It works!
The issue was that even though I tried reinstalling with 
```
pip uninstall numpy wikipedia2vec
pip install numpy==1.19.2
pip install wikipedia2vec
```
It did not work because pip used a previously built wheel - which was, as you said, built with the wrong version of numpy.
I had to replace the last step with 
`pip install wikipedia2vec --no-cache-dir` 
to get a new build, which is working.

Thank you very much for the swift help!
